public/viewjs/metrics.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Metrics Javascript
+ */
+
+/* Charting */
+var dataPoints = [];
+$("#metrics-table tbody tr").each(function () {
+	var self = $(this);
+	var label = self.find("td:eq(0)").attr('data-chart-label');
+	var value = Number(self.find("td:eq(1)").attr('data-chart-value'));
+	var dataPoint = { label: label, y: parseFloat((Math.round(value * 100) / 100).toFixed(2))};
+	dataPoints.push(dataPoint);
+});
+
+var options = {
+	exportEnabled: true,
+	legend:{
+		horizontalAlign: "center",
+		verticalAlign: "bottom"
+	},
+	data: [{
+		type: "pie",
+		showInLegend: true,
+		toolTipContent: "<b>{label}</b>: ${y} (#percent%)",
+		indexLabel: "{label}",
+		legendText: "{label} (#percent%)",
+		indexLabelPlacement: "outside",
+		valueFormatSTringt: "#,##0.##",
```

Comment:
If that's about number formatting, should respect locale as any other place a number is shown ([ref](https://github.com/grocy/grocy/blob/91700e7dae4b35c32c2483ce0837a714df87a398/public/js/grocy.js#L633-L670)).

Reply:
I refactored to the `chartjs` library. We can resolve this one.